src/hardware/memory.cpp

```diff
@@ -57,7 +57,11 @@ static struct MemoryBlock {
 	} a20;
 } memory;
 
-HostPt MemBase;
+#ifndef PAGESIZE
+#define PAGESIZE 4096
+#endif
+
+alignas(PAGESIZE) uint8_t MemBase[MAX_MEMORY*1024*1024];
```

Comment:
Just noticed at the top of these diffs, `mem.h` has:

``` c++
#define MEM_PAGESIZE 4096
```

Would prefer we re-use that and avoid some dupe defines.

(but yeah.. PAGE size is unfortunately very non-DRY :sweat_drops: .. at least we won't add more to it)

``` c++
include/mem.h:#define MEM_PAGESIZE 4096
include/paging.h:#define MEM_PAGE_SIZE	(4096)
src/cpu/core_dyn_x86.cpp:#define PAGESIZE 4096
src/cpu/core_dyn_x86.cpp:#define DYN_PAGE_HASH	(4096>>DYN_HASH_SHIFT)
src/cpu/core_dynrec.cpp:#define PAGESIZE 4096
src/cpu/core_dynrec.cpp:#define DYN_PAGE_HASH	(4096>>DYN_HASH_SHIFT)
src/cpu/dyn_cache.h:#define PAGESIZE_TEMP 4096
src/hardware/memory.cpp:#define MAX_PAGE_ENTRIES (MAX_MEMORY*1024*1024/4096)
src/hardware/vga_memory.cpp:#define VGA_PAGE_A0		(0xA0000/4096)
src/hardware/vga_memory.cpp:#define VGA_PAGE_B0		(0xB0000/4096)
src/hardware/vga_memory.cpp:#define VGA_PAGE_B8		(0xB8000/4096)
src/ints/ems.cpp:#define EMM_PAGEFRAME4K	((EMM_PAGEFRAME*16)/4096)
```